internal/mage/sdk/python.go

```diff
@@ -0,0 +1,87 @@
+package sdk
+
+import (
+	"context"
+	"os"
+
+	"dagger.io/dagger"
+	"github.com/dagger/dagger/internal/mage/util"
+	"github.com/magefile/mage/mg"
+)
+
+var _ SDK = Go{}
+
+type Python mg.Namespace
+
+// Lint lints the Python SDK
+func (t Python) Lint(ctx context.Context) error {
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	_, err = pythonBase(c).
+		Exec(dagger.ContainerExecOpts{
+			Args: []string{"hatch", "run", "lint:style"},
+		}).
+		ExitCode(ctx)
+	return err
+}
+
+// Test tests the Python SDK
+func (t Python) Test(ctx context.Context) error {
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	_, err = pythonBase(c).
+		Exec(dagger.ContainerExecOpts{
+			Args: []string{"hatch", "run", "test"},
+		}).
+		ExitCode(ctx)
+	return err
+}
+
+// Generate re-generates the SDK API
+func (t Python) Generate(ctx context.Context) error {
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	_, err = pythonBase(c).
+		Exec(dagger.ContainerExecOpts{
+			Args: []string{"hatch", "run", "generate"},
+		}).
```

Comment:
You should do `hatch run lint:fmt` after `generate` to keep consistent output.

Reply:
Done!